src/hooks/tests/useDisableSpawnExpirationCheckbox.test.tsx

```diff
@@ -20,6 +20,7 @@ const spruceConfigMock = {
         banner: "",
         ui: {
           userVoice: "https://feedback.mongodb.com/forums/930019-evergreen",
+          defaultProject: "evergreen",
```

Comment:
Just wondering how these `defaultProject` additions relate to schedule tasks stuff